### DISCUSSION TITLE
Handling .Net 4.8 as a pre-requisite

### DISCUSSION THREAD

**Discussion by yuvnith** [wixtoolset/issues#7453] [Questions]
We have developed Wix setups that depend on .Net Framework 4.8. In order to prevent the setup to continue we have used a bal condition to check if 4.8 Net Framework is installed. We have around 100 projects where we have to include this condition.
Is there any better way to handle this pre-requisite, instead of adding this common logic in all the setups?
Can any of you please suggest a better approach(if any)?

**Comment by robmen** [comment]
I'd define it once in a .wixlib, then reference it where needed. This may have already been done in the NetFx.wixext but you'd want to double-check to be sure.
